src/applications/deploying-apps.md

```diff
@@ -63,10 +63,35 @@ Setting                                         | Description
 DEIS_DISABLE_CACHE                              | if set, this will disable the [slugbuilder cache][] (default: not set)
 DEIS_DEPLOY_BATCHES                             | the number of pods to bring up and take down sequentially during a scale (default: number of available nodes)
 DEIS_DEPLOY_TIMEOUT                             | deploy timeout in seconds per deploy batch (default: 120)
+DEIS_DEPLOY_NOW                                 | if set to false, this will disable synchronous deployment (default: no set)
 IMAGE_PULL_POLICY                               | the kubernetes [image pull policy][pull-policy] for application images (default: "IfNotPresent") (allowed values: "Always", "IfNotPresent")
 KUBERNETES_DEPLOYMENTS_REVISION_HISTORY_LIMIT   | how many [revisions][kubernetes-deployment-revision] Kubernetes keeps around of a given Deployment (default: all revisions)
 KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS | how many seconds kubernetes waits for a pod to finish work after a SIGTERM before sending SIGKILL (default: 30)
 
+### Asynchronous Deployment
+
+By default, `git push` and `deis pull` will create a build, a release and proceed to the deployment.
+Starting v2.13 version, it's possible to skip the deployment and trigger the deployment of a release later on.
+
+#### Disabling Synchronous Deployment
+
+There is two way to disable the synchronous deployment:
+
+- Using the command line:
+  - The first one which applies for `git push` and `deis pull` is to set `DEIS_DEPLOY_NOW` to `false` in your application settings.
+  - The second one which only applies for `deis pull`, is to pass `--deploy-now false` option to `deis pull` command.
+- Using the API:
+  - Using the [API](https://deis.com/docs/workflow/reference-guide/controller-api/v2.3/#create-application-build) passing `deploy_now: false` parameter.
```

Comment:
a bit more proper: "Using the API and passing the `deploy_now: false` parameter."